core/src/main/java/io/grpc/internal/BinaryLogProvider.java

```diff
@@ -192,4 +258,29 @@ public InputStream stream(byte[] value) {
       }
     }
   }
+
+  /**
+   * A CallId is two byte[] arrays both of size 8 that uniquely identifies the RPC. Users are
+   * free to use the byte arrays however they see fit.
+   */
+  public static final class CallId {
+    private static final byte[] emptyHi = new byte[8];
+    public final byte[] hi;
```

Comment:
If you're storing a high and low portion, why bother with the `byte[]`? It seems to incur the pain of `byte[]` without the benefits. Why not two `long`s or a single `byte[]`?

I guess it doesn't really matter.